src/IO/H5/Python/ExtractDatFromH5.py

```diff
@@ -76,6 +88,11 @@ def extract_dat_files(filename, out_dir, list=False, force=False):
 @click.argument("out_dir",
                 type=click.Path(file_okay=False, dir_okay=True, writable=True),
                 required=False)
+@click.option('--num-cores',
+              '-j',
+              default=1,
```

Comment:
I'm very hesitant to have the default all available cores because of "head-node etiquette" on clusters. Hopefully you aren't extracting so many big files that you use up a full head-node, but to just avoid this altogether I'd like to keep the default at 1. The user is then responsible for paralleling things.

Reply:
ok 👍